src/kernel/src/fs/host/vnode.rs

```diff
@@ -35,18 +35,16 @@ impl crate::fs::VnodeBackend for VnodeBackend {
 
     fn getattr(self: Arc<Self>, vn: &Arc<Vnode>) -> Result<VnodeAttrs, Box<dyn Errno>> {
         // Get file size.
-        let size = match self.file.len() {
-            Ok(v) => v,
-            Err(e) => return Err(Box::new(GetAttrError::GetSizeFailed(e))),
-        };
+        let size = self.file.len().map_err(GetAttrError::GetSizeFailed)?;
 
         // TODO: Check how the PS4 assign file permissions for exfatfs.
         let mode = match vn.ty() {
             VnodeType::Directory(_) => Mode::new(0o555).unwrap(),
-            VnodeType::Character => unreachable!(), // The character device should only be in the devfs.
+            VnodeType::File | VnodeType::Link => todo!(),
+            VnodeType::Character => unreachable!(), // Character devices should only be in devfs.
         };
 
-        Ok(VnodeAttrs::new(Uid::ROOT, Gid::ROOT, mode, size))
+        Ok(VnodeAttrs::new(Uid::ROOT, Gid::ROOT, mode, size, u32::MAX))
```

Comment:
Does `u32::MAX` as fsid are actually correct here?

Reply:
<img width="273" alt="image" src="https://github.com/obhq/obliteration/assets/48620541/9fdde26a-5543-43e3-b0cc-2a8b4738c4e1">

Reply:
Seems like someone too lazy to implement it correctly here 😂

Reply:
What do you mean? I thought we're doing what FreeBSD does.

Reply:
To be honest, I have no idea what else I should do here

Reply:
I mean someone at Sony are lazy to make implement this correctly. So we have no choice but follow it.

Reply:
Oh. I thought that you meant me :sweat_smile: